src/matrix_validator/validator_polars.py

```diff
@@ -31,55 +47,77 @@ def validate(self, nodes_file_path, edges_file_file_path):
             validate_kg_edges(edges_file_file_path, self.output_format, self.get_report_file())
 
 
+def get_biolink_model_prefix_keys():
+    from . import resources
+
+    with il_resources.open_text(resources, "biolink-model.yaml") as file:
+        bl_model_data = list(yaml.load_all(file, Loader=SafeLoader))
+    return list(bl_model_data[0]["prefixes"].keys())
+
+
+def get_biolink_model_knowledge_level_keys():
+    from . import resources
+
+    with il_resources.open_text(resources, "biolink-model.yaml") as file:
+        bl_model_data = list(yaml.load_all(file, Loader=SafeLoader))
+    return list(bl_model_data[0]["enums"]["KnowledgeLevelEnum"]["permissible_values"].keys())
+
+
+def get_biolink_model_agent_type_keys():
+    from . import resources
+
+    with il_resources.open_text(resources, "biolink-model.yaml") as file:
+        bl_model_data = list(yaml.load_all(file, Loader=SafeLoader))
+    return list(bl_model_data[0]["enums"]["AgentTypeEnum"]["permissible_values"].keys())
```

Comment:
These are also available as pydantic enumerations from 

```
from biolink_model.datamodel.pydanticmodel_v2 import KnowledgeLevelEnum, AgentTypeEnum
[a.value for a in AgentTypeEnum]
[k.value for k in KnowledgeLevelEnum]
```

Not sure

Reply:
I like this approach as I don't like the application to be network dependent (requiring a download of biolink-model.yaml) at runtime or customizing the build to download files.

Reply:
Changed to read from biolink_model.yaml from the biolink-model dependency via importlib